### PR TITLE
Compatibility with tasty 1.5 + CI

### PR DESCRIPTION
**Pull request by sergv**
Looks like there's no CI yet so I propose to add a simple one that should be easy on maintenance.

### CODE REVIEW

**Comment by Bodigrim** [comment]
@ocharles @andreasabel just a very gentle reminder about this.

**Comment by ocharles** [comment]
@Bodigrim thanks. I'm on holiday for a while longer but plan to look at this when I get back

**Comment by ocharles** [comment]
Thanks!

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          haddock: [true]
+        - os: macOS-latest
+          ghc: "9.6"
+          haddock: [false]
+        - os: windows-latest
+          ghc: "9.6"
+          haddock: [false]
+    steps:
+    - uses: actions/checkout@v3
+    - uses: haskell/actions/setup@v2
```
Comment (andreasabel) [inline .github/workflows/ci.yaml:41]: Please switch to `haskell-actions/setup@v2`.

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      id: setup-haskell-cabal
+      with:
+        ghc-version: ${{ matrix.ghc }}
+    - uses: actions/cache@v2
```
Comment (andreasabel) [inline .github/workflows/ci.yaml:45]: Latest is `v3` if I am not mistaken.

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
+name: CI
+on:
+  - push
+  - pull_request
```
Comment (andreasabel) [inline .github/workflows/ci.yaml:4]: Consider restricting the triggers to `master` to avoid duplicate CI runs.

**Review by andreasabel — COMMENTED** [review]
Thanks!

**Pull request merged**